src/makeRobot.js

```diff
@@ -38,7 +38,62 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
якщо змінна дорівнює ключу ти можеш скористатися скороченним записом

```suggestion
    name,
    wheels,
    version,
```